src/ucp/core/ucp_request.inl

```diff
@@ -176,12 +176,23 @@ UCS_PTR_MAP_IMPL(request, 0);
     }
 
 
+#define UCP_REQUEST_CHECK_OPT_FLAGS(_param) \
+    if (ENABLE_PARAMS_CHECK && \
+        ucs_test_all_flags((_param)->op_attr_mask, \
+                           (UCP_OP_ATTR_FLAG_FAST_CMPL | \
```

Comment:
We can use `UCP_PROTO_SELECT_OP_ATTR_MASK` for the combination of the flags, but I thought that maybe in the future this MACRO can hold another flag that can be combined with other opt flags.